widget.go

```diff
@@ -18,6 +18,9 @@ type Widget interface {
 // function and should be exactly one instance per widget in memory.
 type WidgetRenderer interface {
 	// BackgroundColor returns the color that should be used to draw the background of this renderer’s widget.
+	// Deprecated: Widgets will no longer have a background to support hover and selection indication in collection widgets.
+	// If a widget requires a background color or image, this can be achieved by using a canvas.Rect or canvas.Image
+	// as the first child of a MaxLayout, followed by the rest of the widget components.
```

Comment:
`Deprecated:` should start a new paragraph and therefore needs to be separated with an empty line.
```suggestion
	//
	// Deprecated: Widgets will no longer have a background to support hover and selection indication in collection widgets.
	// If a widget requires a background color or image, this can be achieved by using a canvas.Rect or canvas.Image
	// as the first child of a MaxLayout, followed by the rest of the widget components.
```

Reply:
Danke!